Client/Frontend/Autofill/CreditCard/CreditCardBottomSheet/CreditCardBottomSheetHeaderView.swift

```diff
@@ -110,6 +110,7 @@ class CreditCardBottomSheetHeaderView: UITableViewHeaderFooterView, ReusableCell
             mainContainerStackView.topAnchor.constraint(equalTo: topAnchor, constant: 0),
             mainContainerStackView.bottomAnchor.constraint(equalTo: bottomAnchor, constant: -UX.bottomSpacing),
         ])
+        titleLabel.accessibilityFrame = titleLabel.frame.updateWidth(byPercentage: 80)
```

Comment:
This feels like the wrong approach. The titleLabel is overlapping with the close button. This would cause issues with long titles as well. So instead of adjusting the `accessibilityFrame` we should fix the constraints that cause the overlap.

Reply:
adjusting the constraint would cause issues with long titles on iPad